src/main/java/gov/cms/madie/models/measure/Measure.java

```diff
@@ -129,6 +131,8 @@ public class Measure extends ResourceAcl {
   private String cmsId;
   
   private ReviewMetaData reviewMetaData = new ReviewMetaData();
+  
+  private String scoring;
```

Comment:
Does the Scoring field in groups also requires an update to its validation ? Currently, Group.scoring is set to be not null, but for QDM measures that field can be NULL.

Reply:
@RohitKandimalla The Measure level scoring is required only for QDM Measures. As for Group.scoring validation, we could either set it to the Measure.scoring for QDM Measures, or using a different model. So at this point, it's probably ok to leave Group.scoring as is. Thanks for the suggestion though.